### PR TITLE
update-locall-test-flow

### PR DESCRIPTION
**Pull request by jamuszyn**

### CODE REVIEW

File: Products/Brass-Beta/gWASM.md [hunk 1]
```diff
@@ -51,7 +54,289 @@ In this quick tutorial you will
 
 [Follow installation instructions](/Products/Brass-Beta/Installation.md). Remember that it is required to run in the background during gWASM computations.
```
Comment (kubkon) [inline Products/Brass-Beta/gWASM.md]:
```suggestion
[Follow installation instructions](/Products/Brass-Beta/Installation.md). Remember that it is required to run Golem instance in the background during gWASM computations.
```

Comment (kubkon) [reply Products/Brass-Beta/gWASM.md]: Somehow the lack of subject in this sentence made it sound weird.

File: Products/Brass-Beta/gWASM.md [hunk 2]
```diff
@@ -51,7 +54,289 @@ In this quick tutorial you will
...
 
-#### 2. Create a subnet of nodes
+---
+
+#### 2. Prerequisites
+
+Make sure that you have **Rustlang** and **emscripten** installed on your machine. 
+
+* [Rust](https://www.rust-lang.org/tools/install) - rustup toolchain add nightly `rustup target add wasm32-unknown-emscripten --toolchain nightly` TODO [JK - Rust issue link]
```
Comment (kubkon) [inline Products/Brass-Beta/gWASM.md]:
```suggestion
* [Rust](https://www.rust-lang.org/tools/install) - rustup toolchain add nightly `rustup target add wasm32-unknown-emscripten --toolchain nightly` (**NB** it is currently necessary to rely on the `nightly` toolchain due to a bug with the Emscripten target in Rust, which has already been fixed -- see [rust-lang/rust#67976](https://github.com/rust-lang/rust/pull/67976) for reference -- however, won't land in stable for a couple of releases. Once it lands in stable, we'll update the docs. Until then, please use the `nightly` toolchain to build your gWasm apps using Rust.)
```

Comment (jamuszyn) [reply Products/Brass-Beta/gWASM.md]: @nieznanysprawiciel is testing all the steps on Windows. Yesterday it worked on MacOs so it should also work smoothly on Linux systems.

File: Products/Brass-Beta/gWASM.md [hunk 3]
```diff
@@ -39,19 +39,353 @@ If you would like to share your thoughts about **gWASM** please fill out this **
...
+**2.1.1. Installation:**
+
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+
+**2.1.2. Toolchain version:**
+
+```bash
+rustup toolchain add 1.38.0
```
Comment (kubkon) [inline Products/Brass-Beta/gWASM.md]: Hmm, why are we forcing version `1.38.0` here?

Comment (jamuszyn) [reply Products/Brass-Beta/gWASM.md]: By @prekucki recommendation, as this stable of toolchain was tested and works properly with emscripten fastcomp backend `1.38.45-fastcomp`

**Pull request merged**